index.html

```diff
@@ -5,7 +5,7 @@
   <meta charset="UTF-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Document</title>
+  <title>Portfolio setup </title>
```

Comment:
- Hi :clap:, your page title is not descriptive enough. This is your personal portfolio, so it should be descriptive. e.g 'John Does's portfolio'. kindly improve on this.

Reply:
### MAIN HEADER H1, USE H2-H6 FOR SUB-HEADING

- Hi :clap:, the `<h1 class="primary-text">I’m Ankit kush. Glad to see you!</h1>` is the main heading of your page and you should have only one h1 on your page to improve the web accessibility of your page. kindly update ALL THE REST h1 you've used in your project's title and ABOUT SECTION. e.g: `<h1 class="title1">Tonic</h1>`

Reply:
### COLOR CONTRAST

- Hey, let's fix the color contrast we have, see the image below. you can use this [wcag chrome estention](https://chrome.google.com/webstore/detail/wcag-color-contrast-check/plnahcmalebffmaghcpcmpaciebdhgdf) to test and fix your color contrast



![Screenshot from 2023-04-05 14-20-57](https://user-images.githubusercontent.com/59397159/230095094-f13e4071-d694-4bb9-b9d7-c0a64a17aa10.png)
![Screenshot from 2023-04-05 14-20-36](https://user-images.githubusercontent.com/59397159/230095105-5fd078c9-2e4c-461f-a5d7-474fa526b35c.png)
![Screenshot from 2023-04-05 14-19-31](https://user-images.githubusercontent.com/59397159/230095111-5a8e0e17-74b9-4321-8f9d-8311110bfc55.png)